### DISCUSSION TITLE
Could the Famicom Disk System be supported?

### DISCUSSION THREAD

**Discussion by ghost** [sanni/cartreader#506] [Ideas & Suggestions]
Would it be possible to dump Famicom Disk System games? Or are there limitations on the Sanni cart reader that would prevent this?

**Comment by sanni** [comment]
You can find an open source Famicom Disk System dumper here: 
https://github.com/ClusterM/famicom-dumper-writer
https://github.com/ClusterM/famicom-dumper

The second link uses basically the same chip as the Open Source Cartridge Reader, so it seems to be possible for the hardware, maybe it needs some modification though. 
Ofc someone would also need to write/adapt the software/Arduino sketch too.

**Reply by FlitPix** [reply]
It should be noted that this is not specifically an FDS dumper, it is a Famicom dumper that supports FDS. You'll need an FDS to dump Disk System cards.

Regardless, I would also like to express support for this; it is always great to see this project support more systems!

**Comment by RottenToTheGore** [comment]
If you have a working FDS, you can also use the FDSStick

**Reply by ghost** [reply]
I do, but was hoping to not need to buy another piece of hardware.

**Comment by Blood-PawWerewolf** [comment]
+1 for FDS support.

**Comment by ozoramore** [comment]
Hello. I found an interesting post about the Famicom Disk System.
https://x.com/ygawa/status/1794379934665994256
It says that you can extract FDS BIOS with Cartreader. (But I don't have a test environment...)
